lib/rack/param_to_cookie.rb

```diff
@@ -19,6 +19,7 @@ def initialize app, param_cookies
         options[:env_name] ||= param
         options[:ttl] ||= 60*60*24*30 # 30 days
         options[:set_cookie_options] ||= {}
+        options[:max_length] ||= 30 # characters
```

Comment:
I think we should definitely set a default max length, but 30 strikes me as a bit short --- UUIDs, SHA1 hashes, etc., are all longer than this. What would you say to 64 characters?